plugins/analytics-module-ga/src/apis/implementations/AnalyticsApi/GoogleAnalytics.test.ts

```diff
@@ -508,4 +508,63 @@ describe('GoogleAnalytics', () => {
       expect(lastData.queueTime).toBeUndefined();
     });
   });
+
+  describe('api backward compatibility', () => {
+    it('continue working with legacy App category', () => {
+      const api = GoogleAnalytics.fromConfig(basicValidConfig);
+
+      expect(api.captureEvent).toBeDefined();
+
+      api.captureEvent({
+        action: 'navigate',
+        subject: '/',
+        context,
+      });
+
+      const [command, data] = ReactGA.testModeAPI.calls[1];
```

Comment:
I think `ReactGA.testModeAPI.calls[0]` would contain the stuff in context that you could assert on?

Reply:
Aaah, interesting.  Then does `data` contain the category?  Just trying to make sure that the assertions (here, but maybe more importantly below) are testing the compatibility logic as expected.

Reply:
Got it, I added a test here 🙂 : https://github.com/backstage/backstage/pull/22482/commits/c6387c1fe5e7b67bb15b10db494ab65dd7b11901